builtInFunctions/esdtDataStorage.go

```diff
@@ -410,10 +410,7 @@ func (e *esdtDataStorage) setReservedToNilForOldToken(
 		return ErrNilUserAccount
 	}
 	dataOnUserAcc, err := userAcc.AccountDataHandler().RetrieveValue(esdtNFTTokenKey)
```

Comment:
I would use this approach to be more clear that this is not an real error:

```
dataOnUserAcc, errNotCritical := userAcc.AccountDataHandler().RetrieveValue(esdtNFTTokenKey)
shouldIgnoreToken := errNotCritical != nil || len(dataOnUserAcc) == 0
if shouldIgnoreToken {
      return nil
}

```

Reply:
done